step-templates/aws-configure-lambda-api-gateway-integration.json

```diff
@@ -0,0 +1,121 @@
+{
+  "Id": "7e818c42-8c59-4a14-b612-2b9cb69fcf4a",
+  "Name": "Configure Lambda API Gateway Integration",
```

Comment:
Change name to `AWS - Configure Lambda API Gateway Integration` to match others?

Reply:
Name updated